study.md

```diff
@@ -54,5 +58,6 @@ like enumerables?  How are they different?  Please compare and contrast these
 types.
 
 ```md
-<!-- your answer here -->
+difference: enumerables are finite, whereas streams may not be.
```

Comment:
Streams should generally be treated as infinite, even though they can be finite, and enumerables should generally be treated as finite, even though they can be infinite.  An example of an enumerable that's infinite is one that returns a digit of π on each iteration.